src/main/webapp/data/userMapData.json

```diff
@@ -1286,6 +1315,7 @@
       "Metropolitan Community College",
       "MGH Institute of Health Professions",
       "Mid Pacific Institute",
+      "Middle Creek High School",
       "Miami Dade College",
       "Miami International University of Art and Design",
       "Miami University",
```

Comment:
Not your fault, but I found out that `Miami University` and `Miami University of Ohio` (one line below) are the same university. Could you delete `Miami University of Ohio`?

Reply:
Deleted it.